lib/WP_Auth0_Api_Client.php

```diff
@@ -320,10 +321,16 @@ public static function create_client( $domain, $app_token, $name ) {
 				wp_login_url(),
 			),
 
+			// Advanced > Grant Types
 			'grant_types'         => self::get_client_grant_types(),
+
+			// Advanced > OAuth > JsonWebToken Signature Algorithm
 			'jwt_configuration'   => array(
 				'alg' => self::DEFAULT_CLIENT_ALG,
 			),
+
+			// "Use Auth0 to do Single Sign On"
+			'sso'                 => true,
```

Comment:
This option is default on for new applications so just making sure it's on for admins who want to use SSO in the plugin.